javascript/ql/src/experimental/Security/CWE-090/Ldapjs.qll

```diff
@@ -0,0 +1,69 @@
+/**
+ * Provides classes for working with [ldapjs](https://github.com/ldapjs/node-ldapjs) (Client only)
+ */
+
+import javascript
+
+module Ldapjs {
+  string getLdapjsClientDNMethodName() {
+    result = ["add", "bind", "compare", "del", "modify", "modifyDN", "search"]
+  }
```

Comment:
```suggestion
  /**
   * Gets a method name on an LDAPjs client that accepts a DN as the first argument.
   */
  private string getLdapjsClientDNMethodName() {
    result = ["add", "bind", "compare", "del", "modify", "modifyDN", "search"]
  }
```

A public method should have qldoc.  
But this method could actually be private....
Lets do both.